website_sale_hide_price/models/product_template.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.multi
+    def _get_combination_info(
+        self,
+        combination=False,
+        product_id=False,
+        add_qty=1,
+        pricelist=False,
+        parent_combination=False,
+        only_template=False
+    ):
+        """Avoid show prices in other website places.
+        This will show 0.00 value"""
+        ret = super()._get_combination_info(
```

Comment:
Out of curiosity, why ret?

Reply:
ret = return ... 
maybe better "res = result"?

Reply:
ok ok ... thanks... I imagined that